engine/lib/behaviors/invalidQuery.js

```diff
@@ -0,0 +1,22 @@
+'use strict'
+const Behavior = require("./behavior")
+const MySQLRepository = require('../mySQLRepository');
+
+class InvalidQuery extends Behavior {
+    constructor(value) {
+        super("INVALID-QUERY", value)
+    }
+
+    async execute() {
+        super.execute()
+        const repository = MySQLRepository.getInstance()
```

Comment:
can this be injected into the class? that would make it easier to test.

Reply:
I was trying to think of a good way to do that, maybe have the factory handle passing it in. I'll give that a try